src/Packetery/Module/Order/Repository.php

```diff
@@ -520,6 +516,37 @@ public function countOrdersToPrint(): int {
 		);
 	}
 
+	/**
+	 * Counts carrier orders.
+	 *
+	 * @param string      $carrierId       Carrier ID.
+	 * @param string|null $shippingCountry Country.
+	 *
+	 * @return int
+	 */
+	public function countOrders( string $carrierId, ?string $shippingCountry ): int {
+		$wpdb = $this->wpdb;
+
+		$join = [ 'JOIN `' . $wpdb->posts . '` wp_p ON wp_p.`ID` = o.`id`' ];
```

Comment:
Všechny názvy sloupců a tabulek prosím uzavři do backticks, máme to tak v celém zbytku pluginu.